spec/numberformat.html

```diff
@@ -35,6 +35,9 @@ <h1>Intl.NumberFormat ( [ _locales_ [ , _options_ ] ] )</h1>
         1. Let _style_ be _numberFormat_.[[Style]].
         1. If _style_ is *"currency"*, then
           1. Let _currency_ be _numberFormat_.[[Currency]].
```

Comment:
Why is the line " Let _currency_ be _numberFormat_.[[Currency]]." not inside the same if block of 
```
          1. Let _cDigits_ be CurrencyDigits(_currency_).
          1. Let _mnfdDefault_ be _cDigits_.
          1. Let _mxfdDefault_ be _cDigits_.
```
Nowhere else need the value of _currency_ in this AO, right?